frontend/src/components/gallery/artworkCard/artworkCard.tsx

```diff
@@ -56,6 +57,8 @@ class ArtworkCard extends React.Component<ArtworkCardProps, ArtworkCardState> {
     }
 
     componentDidMount() {
+        // TODO: Is this the right place to set the state?
+        this.setState({inViewport: true});
```

Comment:
That's the initial state of the component right? Then I think it should normally be in the constructor with all the other states. And setting the state field should probably also be inside the constructor as well.

Reply:
It's not intended to be the initial state; this should be the state once we know it's loading. Though, because now it's on the homepage, it's basically the initial state.

You think it might be ok to leave it here if our fall back is to have a gallery page? In that case, the initial state would actually be false until someone goes to the gallery page.

Reply:
You mean lazy loading the component? I think that should be fine then.